.github/workflows/ci.yaml

```
@@ -60,6 +60,37 @@
+        env:
+          ARCHIVE: $\{{ steps.metadata.outputs.archive }}
+          PLUGIN_ID: $\{{ steps.metadata.outputs.plugin-id }}
+
+      - name: Check plugin.json
+        run: |
+          docker run --pull=always \
+            -v $PWD/${ARCHIVE}:/archive.zip \
+            grafana/plugin-validator-cli -analyzer=metadatavalid /archive.zip
+        env:
+          ARCHIVE: $\{{ steps.metadata.outputs.archive }}
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)